src/style/index.less

```diff
@@ -93,4 +93,11 @@ content-region {
 	& > main {
 		padding-top: 56px;
 	}
+	
+	display: flex;
+	flex-direction: column;
+}
+
+main {
```

Comment:
I tested and the only thing I noticed was that I think this block needs to be moved to `L93` and the selector changed to:

```css
& > main, & > footer {
```